lib/svc/method/net-physical

```diff
@@ -530,6 +533,15 @@ if smf_is_globalzone; then
     # Make any mtu adjustments that may be necessary
     setup_mtu
 
+    # Create overlay_rules.json
+    config_overlays="$(declare -p | grep "\ CONFIG_overlay_[a-zA-Z0-9_]*=" | cut -d "_" -f 2- | cut -d "=" -f 1)"
+    overlay_rules="{"
+    for overlay_name in ${config_overlays}; do
+        eval "overlay_rule=\${CONFIG_${overlay_name}}"
+        overlay_rules="${overlay_rules}\n    \"${overlay_name}\": \"${overlay_rule}\","
+    done
+    echo -e "$(echo -e "${overlay_rules}" | sed '$s/,$//')\n}" > "${OVERLAY_RULES}"
```

Comment:
Don't create empty overlay_rules.json.

Reply:
Why? Since the file is always a json - an empty json object makes sense from application perspective.

Reply:
We should not reserve the only right to edit `overlay_rules.json`. If there are other apps that edit the file, we don't want to mess their work.